modules/network/src/main/java/org/apache/ignite/network/scalecube/ScaleCubeClusterServiceFactory.java

```diff
@@ -17,18 +17,18 @@
 
 package org.apache.ignite.network.scalecube;
 
-import java.util.List;
-import java.util.stream.Collectors;
 import io.scalecube.cluster.ClusterImpl;
 import io.scalecube.cluster.ClusterMessageHandler;
 import io.scalecube.cluster.membership.MembershipEvent;
 import io.scalecube.cluster.transport.api.Message;
 import io.scalecube.net.Address;
-import org.apache.ignite.network.ClusterService;
-import org.apache.ignite.network.NetworkConfigurationException;
+import java.util.List;
```

Comment:
Looks like either my IDEA settings are incorrect or yours. Mine always puts `java.*` imports on top. Is it wrong?

Reply:
Dunno, I can see both import orders in project